src/blocks/card.scss

```diff
@@ -0,0 +1,129 @@
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 944px;
+  padding: 50px 0;
+  margin: 0 auto;
+}
+
+.card {
+  display: inline-flex;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-bottom: 48px;
+
+  &__item {
+    margin: 0;
+    padding: 0;
```

Comment:
redundant. div has not default magin and padding.
also &__item redundant element
enough .card